kiwi_lint/field_label.py

```diff
@@ -0,0 +1,27 @@
+from pylint import checkers
+from pylint import interfaces
+
+
+class FormFieldLabelChecker(checkers.BaseChecker):
+    """
+    Warns against defining labels in Form Field class constructors.
+    Labels must be placed in the HTML.
+    """
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'form-field-label-checker'
+
+    msgs = {
+        'R4811': ('Do not use label in form Field class constructor.',
+                  'form-field-label-used',
+                  'Do not use label in form Field class constructor,'
+                  'place template label in HTML. See:'
+                  'https://github.com/kiwitcms/Kiwi/issues/738')
+    }
+
+    def visit_call(self, node):
+        if node.func.as_string().endswith('Field') and node.keywords:
+            for keyword in node.keywords:
+                if keyword.arg == 'label':
+                    self.add_message('form-field-label-used', node=node)
+                    break
```

Comment:
Can you also add a similar check for `help_text` attribute b/c I see it in testplans/forms.py and only registration_form.html seems to use it.

Note to self: model fields also have `help_text` attribute which also goes into migration files. We use some of them (e.g. BugSystem) in admin but mostly we don't. They probably need a separate checker.